stable/grc/templates/grc-policy-addon-clusterrole.yaml

```diff
@@ -122,8 +122,8 @@ rules:
   resourceNames:
   - cert-policy-controller
   - config-policy-controller
-  - iam-policy-controller
   - governance-policy-framework
+  - iam-policy-controller
```

Comment:
This order change is because I just copied directly what kubebuilder generated.

Reply:
...but also alphabetical and awesome.